### PR TITLE
feat: created crypto markets option

### PR DESCRIPTION
**Pull request by user2745**
# Description
Mutes crypto markets to prepare for future listings

Fixes Jira Ticket # 1273

## Type of change
Please delete options that are not relevant.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Is this a UI Change? If so please include screenshot of before and after states below

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

# Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have assigned 2 reviewers to check my work
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: lib/crypto_markets/view/crypto_markets.dart [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+      child: ListView.builder(
+        padding: const EdgeInsets.only(top: 10),
+        physics: const BouncingScrollPhysics(),
+        itemCount: 10,
+        itemBuilder: (context, index) {
+          /// CryptoDesktopMarketsCard()
+        },
+      ),
+    );
+  }
```
Comment (rbharath99) [inline lib/crypto_markets/view/crypto_markets.dart:26]: [This has been pointed out so many times do not return the widget](https://dcm.dev/docs/rules/flutter/avoid-returning-widgets/#:~:text=Warns%20when%20a%20method%2C%20function,processor%20time%20for%20the%20operations.) . Please Stop doing that.

File: lib/crypto_markets/view/crypto_markets.dart [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+      child: ListView.builder(
+        padding: const EdgeInsets.only(top: 10),
+        physics: const BouncingScrollPhysics(),
+        itemCount: 10,
+        itemBuilder: (context, index) {
+          /// CryptoMobileMarketsCard()
+        },
+      ),
+    );
+  }
```
Comment (rbharath99) [inline lib/crypto_markets/view/crypto_markets.dart:40]: Same issue here.

**Review by rbharath99 — CHANGES_REQUESTED** [review]

File: lib/crypto_markets/view/crypto_markets.dart (hunk 2 above)
Comment (user2745) [reply lib/crypto_markets/view/crypto_markets.dart:40]: sorry about that, I'll get on it

File: lib/crypto_markets/view/crypto_markets.dart (hunk 1 above)
Comment (user2745) [reply lib/crypto_markets/view/crypto_markets.dart:26]: oops my fault

**Review by rbharath99 — APPROVED** [review]

**Pull request merged**